src/App.js

```diff
@@ -14,11 +17,91 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goodsData: [...goodsFromServer],
```

Comment:
```suggestion
    goods: [...goodsFromServer],
```
название массива принято давать во множественном числе